### ISSUE TITLE
'node-gyp configure build && node install.js ' fails on osx mavericks

### ISSUE THREAD

**Issue by gaultier** [quartzjer/ursa#43]
Hi, osx mavericks user here, I got this error :

```
phil@air-de-phil ~/ursa (master●)$ npm install ursa -g
info trying registry request attempt 1 at 19:49:28
http GET https://registry.npmjs.org/ursa
http 304 https://registry.npmjs.org/ursa

> ursa@0.8.0 install /usr/local/lib/node_modules/ursa
> node-gyp configure build && node install.js

  CXX(target) Release/obj.target/ursaNative/src/ursaNative.o
In file included from ../src/ursaNative.cc:3:
../src/ursaNative.h:6:9: warning: 'BUILDING_NODE_EXTENSION' macro redefined
#define BUILDING_NODE_EXTENSION
        ^
<command line>:4:9: note: previous definition is here
#define BUILDING_NODE_EXTENSION 1
        ^
In file included from ../src/ursaNative.cc:3:
../src/ursaNative.h:7:10: fatal error: 'node.h' file not found
#include <node.h>
         ^
1 warning and 1 error generated.
make: *** [Release/obj.target/ursaNative/src/ursaNative.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:789:12)
gyp ERR! System Darwin 13.0.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /usr/local/lib/node_modules/ursa
gyp ERR! node -v v0.10.18
gyp ERR! node-gyp -v v0.12.2
gyp ERR! not ok
npm ERR! ursa@0.8.0 install: `node-gyp configure build && node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ursa@0.8.0 install script.
npm ERR! This is most likely a problem with the ursa package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp configure build && node install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls ursa
npm ERR! There is likely additional logging output above.

npm ERR! System Darwin 13.0.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "ursa" "-g"
npm ERR! cwd /Users/phil/ursa
npm ERR! node -v v0.10.18
npm ERR! npm -v 1.3.25
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/phil/ursa/npm-debug.log
npm ERR! not ok code 0
```

tl;dr:

```
In file included from ../src/ursaNative.cc:3:
../src/ursaNative.h:7:10: fatal error: 'node.h' file not found
#include <node.h>
         ^
```
